packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -605,12 +617,16 @@ export class ModularChangeFamily
 	}
 }
 
-function revisionIndexerFromInfo(revInfos: readonly RevisionInfo[]): RevisionIndexer {
-	return (tag: RevisionTag): number => {
+function revisionHelperFromInfo(revInfos: readonly RevisionInfo[]): RevisionHelper {
```

Comment:
Since `RevisionHelper` is only used to look up revision related information, maybe we could give it a more descriptive name. Maybe something like `RevisionMetadataSource`?